### ISSUE TITLE
Multiple errors from a single validator

### ISSUE THREAD

**Issue by ThePenguin1140** [ngx-formly/ngx-formly#2378]
**Is your feature request related to a problem? Please describe.**
As a developer I would like to validate multiple fields with a single validator. This would allow me to perform server-side, asynchronous validation for an entire form section but still display distinct error messages for each field.

**Describe the solution you'd like**
I would like to be able to specify more message + errorPath combinations in a single validator return objects.
Perhaps: 

```ts
{
  groupValidator: [
  {
    errorPath: 'password',
    message: 'Please choose a stronger password!', 
  },
  {
    errorPath: 'username',
    message: 'This username is already taken!',
  },
  ]
}
```
**Describe alternatives you've considered**
The validation error return object of a validator could also be an array or it could be an object with multiple named validators.
For example: 

```ts
{
  passwordValidator: {
    errorPath: 'password',
    message: 'Please choose a stronger password!', 
  },
  usernameValidator: {
    errorPath: 'username',
    message: 'This username is already taken!',
  },
}
```

**Additional context**

**Comment by ThePenguin1140** [comment]
Looks great! Thank you for the quick implementation 👍 

Will this be a part of `v5` or are you planning on keeping this for `next`?

**Comment by aitboudad** [comment]
it will be part of `v5`

**Comment by aitboudad** [comment]
Hi, the fix is now included in the dev version `5.10.0-dev.ad26817d` which you may use until the stable release `5.10.0` is available (scheduled for Saturday).

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.10.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
